src/test/scala/dotty/semanticdb/Tests.scala

```diff
@@ -156,7 +156,7 @@ class Tests {
   def assertNoDiff(obtained: String, expected: String): Unit = {
     if (obtained.isEmpty && !expected.isEmpty) fail("obtained empty output")
     def splitLines(string: String): java.util.List[String] =
-      string.trim.replace("\r\n", "\n").split("\n").toSeq.asJava
+      string.trim.replace("\r\n", "\n").split("\n").map(_.nn).toSeq.asJava
```

Comment:
A Java method/field that would need a more precise type annotation than just null/non-null at the outermost level.